tests/python/unittest/test_tir_schedule_cache_read_write.py

```diff
@@ -414,15 +432,15 @@ def cache_read_multi_consumer_target() -> None:
             with T.block("A"):
                 vi = T.axis.S(128, i * 16 + j)
                 A[vi] = 1.0
-        for j in T.grid(16):
-            with T.block("A"):
-                vi = T.axis.S(128, i * 16 + j)
-                A_global[vi] = A[vi]
         for j in T.grid(16):
             with T.block("B"):
                 vi = T.axis.S(128, i * 16 + j)
                 B[vi] = A[vi] + 1.0
 
+    for i in T.grid(128):
```

Comment:
Why do we need to change this testcase? A common practice is not to change the existing test case if possible.

Reply:
Thanks, indeed. but if considering a case that R-W-R a same buffer, and we want to _**cache_read**_ the second R, the 1st R should be ignored. This PR's solution is to specify the 2nd R in _**consumer_blocks**_ and ignore R if it is not in _**consumer_blocks**_. So for this test case, block("B") is ignored. Since the purpose of this case is to cache_read block("C"), the cache block seems to be more reasonable if next to block("C"). Any better solutions is welcomed.

Reply:
cc @jwfromm since he is the author of the test case